packages/h5wasm/src/h5wasm-api.test.ts

```diff
@@ -25,7 +25,7 @@ async function loadTestFile(): Promise<ArrayBuffer> {
   return resp.arrayBuffer();
 }
 
-test('test file matches snapshot', async () => {
+test.skip('test file matches snapshot', async () => {
```

Comment:
Since yesterday, the `H5WasmApi` test snapshot seems to be different on my machine and on the CI. Some of the `UInt8Array`s that contain memory addresses have slightly different bytes... This is unrelated to this PR (`main` is failing as well), so I'm skipping the test until I can figure out a solution.